tests/system/providers/amazon/aws/example_emr_serverless.py

```diff
@@ -108,6 +109,15 @@
         job_run_id=start_job.output,
     )
     # [END howto_sensor_emr_serverless_job]
+    wait_for_job.poke_interval = 10
+
+    # [START howto_operator_emr_serverless_stop_application]
```

Comment:
Please update doc as well and use these new tags `howto_operator_emr_serverless_stop_application` to include examples in the docs

Reply:
oh yes right